web/src/styles/Alert.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+import { Box } from 'rebass';
+
+export const alertColors = {
+  success: ['#e7f4e4', '#2e8540'],
+  warning: ['#fff1d2', '#fdb81e'],
+  error: ['#f9dede', '#e31c3d'],
+  info: ['#e1f3f8', '#02bfe7'],
+  fallback: ['#f1f1f1', '#8b8b8b']
+};
+
+const cssRules = kind => {
+  const colors = alertColors[kind] || alertColors.fallback;
+
+  return `
+    background-color: ${colors[0]};
+    border-left: 0.5rem solid ${colors[1]};
+  `;
+};
+
+const Alert = styled(Box)`
+  ${props => cssRules(props.kind)};
+  padding: 1rem;
+`;
+
+export default Alert;
```

Comment:
Oh, I see.  So `<Alert>` is used less like a standard component and more like a style class?  That's interesting!

Reply:
Yes! I think there are two levels of hierarchy and components we want to support -- these style class components, then one level up shared components that take in content / data props and compose slightly bigger things (using those style class components from below)